bundles/binding/org.openhab.binding.myq/src/main/java/org/openhab/binding/myq/internal/MyqBinding.java

```diff
@@ -316,6 +316,19 @@ private void computeCommandForItem(Command command, String itemName) {
                 } else if (command.equals(OnOffType.OFF) || command.equals(UpDownType.DOWN)) {
                     myqOnlineData.executeGarageDoorCommand(garageopener.getDeviceId(), 0);
                     beginRapidPoll(true);
+                } else if (command instanceof StringType) {
+                    String stringValue = ((StringType) command).toString();
+                    if (stringValue.equalsIgnoreCase(GarageDoorStatus.OPEN.getLabel())
+                            || stringValue.equalsIgnoreCase(GarageDoorStatus.OPENING.getLabel())) {
+                        myqOnlineData.executeGarageDoorCommand(garageopener.getDeviceId(), 1);
+                        beginRapidPoll(true);
+                    } else if (stringValue.equalsIgnoreCase(GarageDoorStatus.CLOSED.getLabel())
+                            || stringValue.equalsIgnoreCase(GarageDoorStatus.CLOSING.getLabel())) {
+                        myqOnlineData.executeGarageDoorCommand(garageopener.getDeviceId(), 0);
+                        beginRapidPoll(true);
+                    } else {
+                        logger.warn("Unknown string command {}", stringValue);
+                    }
```

Comment:
To command a door to "Closing" or "Closed" or "Opening" doesn't make sense as they all describe the current state, not the verb to "close" or "open" the door.  There must be other ways to create a good sitemap, since the button labels can be set to whatever you want.  

But since there is no harm in having several strings all mean the "open" verb, and several strings all mean the "close" verb, perhaps just add PercentType.ZERO, "Close", "OFF" and "0" to your list of "close" synonyms, and add PercentType.HUNDRED, "ON" and "1" to your list of "open" synonyms?  ("Open" is already there because it's both an adjective and a verb in English).